source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0026
+
+<b>Functional Description</b>
+
+This D.G./D.I. can be used only  <br/>         - in case of Export followed by transit (maximum one MRN included per one House Consignment) <br/>     AND <br/>         - if the &lt;TRANSIT OPERATION.Additional Declaration Type&gt; is EQUAL to 'A'.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0029
+
+<b>Functional Description</b>
+
+The D.I. will be filled in with new value amending initial declaration in case of incident. In case goods were initially not containerized and are placed in a container, or the initial container is replaced by another container then &lt;CONSIGNMENT-INCIDENT-TRANSHIPMENT.Container indicator&gt; is equal to '1' else &lt;CONSIGNMENT-INCIDENT-TRANSHIPMENT.Container indicator&gt; is equal to '0'.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0030
+
+<b>Functional Description</b>
+
+If a consignment is moving from one Member State to another Member State via a third country which is not in set of CL009 (CountryCodesCommonTransit) then a &lt;CUSTOMS OFFICE OF TRANSIT (DECLARED)&gt; shall be declared and located in the specific Member States. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0032
+
+<b>Functional Description</b>
+
+This CUSTOMS OFFICE can be the DECLARED Office or the ACTUAL Office.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0033
+
+<b>Functional Description</b>
+
+The Data Item &lt;AUTHORISATION.Reference number&gt; must be valid in CDMS or in the National Decision Management System.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0034
+
+<b>Functional Description</b>
+
+In case of Export followed by Transit and whenever the internal transit procedure is applied, the Declared Office of Destination needs to be ‘appropriate’, otherwise the initial submission and/or subsequent amendment requests of the transit declaration data as submitted by the Holder of the Transit Procedure to the Office of Departure has to be rejected. This can be validated as follows:<br/>A/ In case the Declared Office of Destination belongs to EU MS (CL010- CountryCodesCommunity), and its Custom Office Reference Number is included in both CL172- CustomsOfficeDestination and CL294-CustomsOfficeExitDeclared, then it is considered ‘appropriate’ (otherwise is considered not ‘appropriate’);<br/>B/ In case the Declared Office of Destination belongs to CTC (CL112- CountryCodesCTC), it is considered by default ‘appropriate’.<br/>When the Declared Office of Destination is considered as not ‘appropriate’, the messages CC013C or CC015C will be responded with CC056C that will report the error code '12- Codelist violation'.<br/>
+
+<b>Technical Description</b>
+
+N/A
+
+##G0035
+
+<b>Functional Description</b>
+
+The Holders of the Transit Procedure can only request information on their own Guarantee[s].
+
```

Comment:
Grammar/style issues - 
- Positioning of `only`.
- Why are square brackets used in `Guarantee(s)`. Parentheses are customary in plain text.

The following rule is not mentioned in GDS style guide but should be followed at all times: **Only goes only before the word it modifies.**

Suggestion: The Holders of the Transit Procedure can request information only about their own Guarantee(s).